evap/student/views.py

```diff
@@ -65,42 +65,45 @@ def vote_preview(request, course, for_rendering_in_modal=False):
     return render(request, "student_vote.html", template_data)
 
 
+def render_vote_page(request, course):
```

Comment:
What do you think about a name like `get_valid_form_groups_or_render_vote_page`? It took me a while to understand the `if all(all(...))` line, I think this name change could help.